streamlit-chatbot-app/app.py

```diff
@@ -0,0 +1,99 @@
+import streamlit as st
+import os
+# If using OpenAI API
+from openai import OpenAI 
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service.serving import ChatMessage, ChatMessageRole
+
+# Initialize the Databricks Workspace Client
+w = WorkspaceClient()
+
+def get_user_info():
+    headers = st.context.headers
+    return dict(
+        user_name=headers.get("X-Forwarded-Preferred-Username"),
+        user_email=headers.get("X-Forwarded-Email"),
+        user_id=headers.get("X-Forwarded-User"),
+        access_token=headers.get("X-Forwarded-Access-Token")
+    )
+
+user_info = get_user_info()
+
+model_mapping = {
+    "DBRX Instruct": "databricks-dbrx-instruct",
+    "Meta Llama 3.1 70B Instruct": "databricks-meta-llama-3-1-70b-instruct",
+    "Mixtral-8x7B Instruct": "databricks-mixtral-8x7b-instruct",
+    "Your Own Model": os.getenv("SERVING_ENDPOINT")
```

Comment:
Maybe add an assertion for this environment variable and a corresponding error message?